### PR TITLE
Add Hatsune Miku: Project Mirai  DX

### PR DESCRIPTION
**Pull request by bronozoj**
Adds Hatsune Miku: Project Mirai  DX.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/hatsune-miku-project-mirai-dx/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Hatsune Miku: Project Mirai  DX"
+description = "Hatsune Miku: Project Mirai DX is a 3D rhythm action game starring the world's most famous virtual singer Hatsune Miku. Join the Miku phenomenon by expressing yourself through fun creative tools, sharing your cute customizations, and experiencing some of the best music from the world’s favorite digital diva!\n\nPublisher: Sega\nGenre: Rhythm"
```
Comment (Schplee) [inline games/hatsune-miku-project-mirai-dx/game.dat]: Please remove the line breaks `\n`

File: games/hatsune-miku-project-mirai-dx/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Hatsune Miku: Project Mirai  DX"
```
Comment (Schplee) [inline games/hatsune-miku-project-mirai-dx/game.dat]: `Mirai  DX` There are two spaces between Mirai and DX. After this fix, your PR should be able to be merged 😄

File: games/hatsune-miku-project-mirai-dx/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+description = "Hatsune Miku: Project Mirai DX is a 3D rhythm action game starring the world's most famous virtual singer Hatsune Miku. Join the Miku phenomenon by expressing yourself through fun creative tools, sharing your cute customizations, and experiencing some of the best music from the world’s favorite digital diva! Publisher: Sega Genre: Rhythm"
```
Comment (TheKoopaKingdom) [inline games/hatsune-miku-project-mirai-dx/game.dat]: The `Publisher: Sega Genre: Rhythm` part doesn't really belong in the game description.

File: games/hatsune-miku-project-mirai-dx/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000148C00"
+region = "usa"
+release_date = "2015-09-08"
+
+[[testcases]]
+title = "0004000000148C00"
+compatibility = "0"
```
Comment (TheKoopaKingdom) [inline games/hatsune-miku-project-mirai-dx/game.dat]: According to your wiki article, the game has minor graphical issues, and minor audio issues, so the compatibility should be `1` for *Great*.

**Comment by Schplee** [comment]
LGTM

**Pull request merged**